store/sqlstore/post_store.go

```diff
@@ -905,6 +905,36 @@ func (s *SqlPostStore) GetPostsSince(options model.GetPostsSinceOptions, allowFr
 	return list, nil
 }
 
+func (s *SqlPostStore) GetPostsByUserInChannelSince(options model.GetPostsSinceOptions, userId string) ([]*model.Post, error) {
+
+	var posts []*model.Post
+
+	query := `SELECT
```

Comment:
We can directly check for the message type in the query itself and return a true/false result.

`select exists (Select 1 from posts where updateAt >= <time> and channelID = <> and userID = <> and type = model.POST_AUTO_RESPONDER limit 1)`